### DISCUSSION TITLE
Clarification on custom filters documentation

### DISCUSSION THREAD

**Discussion by james-yin** [thunderclient/thunder-client-support#1302] [Q&A]
In the "Custom Filters" documentation section [https://github.com/rangav/thunder-client-support/blob/master/docs/filters.md#custom-filters](url), the following picture has `message` as part of
`{{message |  appendString("Custom")}}`

So what is this `message`? I failed to see any references to it. Please clarify, thanks.

![image](https://github.com/rangav/thunder-client-support/assets/7233400/fe948c51-a565-41fa-afa8-287a8e2a1e19)

**Comment by rangav** [comment]
Hi @james-yin 

- `message` is env variable 
- You can use filters to modify env variable values
